db/migrate/20220118221746_create_feeds.rb

```diff
@@ -0,0 +1,43 @@
+class CreateFeeds < ActiveRecord::Migration
+  def change
+    remove_column :podcasts, :max_episodes, :integer
+
+    create_table :feeds do |t|
+      t.references :podcast, index: true, foreign_key: true
+      t.string :label
+      t.string :slug, null: false
+
+      # optional overrides
+      t.text :title
+      t.string :url
+      t.string :path
+      t.string :enclosure_prefix
+      t.string :enclosure_template
+      t.integer :display_episodes_count
+      t.integer :display_full_episodes_count
+
+      # optional sub-feed settings
+      t.boolean :private, default: true
+      t.integer :publish_offset_seconds
+      t.text :reject_zones
+      t.text :filter_tags
+      t.text :audio_format
```

Comment:
Some hand waving here - but I imagine this will be a serialized Dovetail arrangement "audio format". Which in arrangement json, looks something like:

```json
{
  "version": 2,
  "data": {
    "f": [
      "files"
    ],
    "t": "aooi",
    "a": {
      "f": "mp3",
      "b": 80,
      "c": 1,
      "s": 44100
    }
  }
}
```